EIPS/eip-5732.md

```diff
@@ -83,6 +84,7 @@ contract SomeBetterToken is ISimpleToken, IERC_COMMIT {
     mapping(address => bytes32) lastCommits;
```

Comment:
L47 has typo: `making room` instead of`making rooms`.

Reply:
Done